### DISCUSSION TITLE
Issue when running the project

### DISCUSSION THREAD

**Discussion by SomeshShinde24** [Vikash-8090-Yadav/Future.WebNet#150] [Q&A]
I am not able to run the project I think because of Localhost 8080, So anyone know the solution ?

**Comment by Vikash-8090-Yadav** [comment]
@SomeshShinde24 can u please share the  project name ot the Direcotry ?

**Reply by SomeshShinde24** [reply]
web3.0-world

**Reply by Vikash-8090-Yadav** [reply]
Ya but can u share the link  of the project ?  As the Web3 World is   the Directory  name and it's not a poject . It is a directory .

**Reply by Vikash-8090-Yadav** [reply]
@SomeshShinde24 are u talking about this issue https://github.com/Vikash-8090-Yadav/Web3.0-World/issues/2 ?
